src/components/PartnerInfo.vue

```diff
@@ -14,7 +14,6 @@
           variant="text"
           icon
           class="close-icon"
-          :size="36"
```

Comment:
rollback size

Reply:
wdym? the size prop is in v-icon tag

Reply:
this is the of the button not of the icon